### PR TITLE
Status Chart: Implement Average PR Age

### PR DESCRIPTION
**Pull request by StephanTLavavej**
[Requested](https://github.com/microsoft/STL/pull/1094#issuecomment-664313688) by @miscco and [suggested](https://github.com/microsoft/STL/issues/921#issuecomment-658064093) by @AlexGuteniev.

As I mentioned in #1043, I prefer combined age as a metric, but this way everyone can see their preferred metrics as the chart is interactive.

Live preview: [stephantlavavej.github.io/STL/](https://stephantlavavej.github.io/STL/)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -211,22 +218,34 @@
...
+                    },
                     {
                         type: 'linear',
                         display: 'auto',
                         id: 'rightAxis',
                         position: 'right',
                         scaleLabel: {
                             display: true,
-                            labelString: 'PR-months',
+                            labelString: 'Combined Age (PR-months)',
```
Comment (BillyONeal) [inline index.html:248]: I can't decide if this scale is pessimistic or optimistic XD

Comment (CaseyCarter) [reply index.html:248]: It's easy to come back later and change to PR-millenia.

File: gather_stats.js [hunk 2]
```diff
@@ -187,6 +187,9 @@ function write_daily_table(script_start, all_prs, all_issues) {
                 }
             }
 
+            const age_avg_days = num_pr === 0 ? 0 : combined_pr_age.as('days') / num_pr;
```
Comment (StephanTLavavej) [inline gather_stats.js:190]: > `num_pr === 0`

A dev can dream...

File: index.html (hunk 1 above)
Comment (StephanTLavavej) [reply index.html:248]: With ~30 open PRs, we accumulate one PR-month per day. :scream_cat:

**Pull request merged**